routes.go

```diff
@@ -5,8 +5,7 @@ import (
 	"net/http"
 )
 
-// Route is an object that defines how a url is called and what function
-// handles the response.
+// Each Route object defines how a url is called and its handler function
```

Comment:
This is a superficial change that breaks godoc. The comment string on a function is supposed to start with the name a function.